src/clients/cli.rs

```diff
@@ -23,26 +24,22 @@ pub struct Cli {
 }
 
 impl Cli {
-    pub fn run<I: Image>(&self, image: I) -> Container<'_, I> {
-        self.run_with_args(image, RunArgs::default())
-    }
-
-    pub fn run_with_args<I: Image>(&self, image: I, run_args: RunArgs) -> Container<'_, I> {
-        let mut docker = self.inner.command();
+    pub fn run<I: Image>(&self, image: impl Into<RunnableImage<I>>) -> Container<'_, I> {
+        let runnable = image.into();
```

Comment:
Nit: If we would shadow the `image` variable here, the diff would be a lot less noisy. Also, I think `image` is still the primary concept of `RunnableImage` contrary to `runnable`.